shenyu-client/shenyu-client-http/shenyu-client-springcloud/src/main/java/org/apache/shenyu/client/springcloud/init/SpringCloudClientBeanListener.java

```diff
@@ -46,21 +46,22 @@
 import java.lang.reflect.Method;
 import java.util.ArrayList;
 import java.util.List;
+import java.util.Map;
 import java.util.Objects;
 import java.util.Optional;
 import java.util.Properties;
 
 /**
- * The type Shenyu client bean post processor.
+ * The type Shenyu client bean listener.
  */
-public class SpringCloudClientBeanPostProcessor implements BeanPostProcessor {
+public class SpringCloudClientBeanListener implements ApplicationListener<ContextRefreshedEvent> {
```

Comment:
please rename to `SpringCloudClientEventListener`.

Reply:
ok, i renamed.